src/pages/library/Datasets.js

```diff
@@ -286,6 +289,39 @@ const framingham = () => h(Participant, {
   }, ['Browse data'])
 ])
 
+const gp2 = () => h(Participant, {
+  logo: { src: gp2Logo, alt: `GP2 logo`, height: '40%' },
+  modalLogoHeight: 100,
+  title: `GP2`,
+  shortDescription: `The Global Parkinson's Genetics Program (GP2) aims to transform our understanding 
+    of the genetic basis of Parkinson’s disease (PD) across diverse populations, including those 
+    underserved in biomedical...`,
+  description: h(Fragment, [
+    p([
+      `The Global Parkinson's Genetics Program (GP2) aims to transform our understanding of the genetic 
+    basis of Parkinson’s disease (PD) across diverse populations, including those underserved in 
+    biomedical research. This program includes targeted collection and analysis of data and samples 
+    from Africa, Asia, Europe and the Americas toward genetic insights that will broaden biomarker and 
+    therapeutic discovery and development. GP2 is a resource of the Aligning Science Across 
+    Parkinson’s (ASAP) initiative, which has developed an ambitious roadmap to tackle key scientific 
+    challenges in PD by supporting meaningful, multidisciplinary collaboration; generating 
+    research-enabling resources; and sharing data.`
+    ])
+  ])
```

Comment:
This shouldn't need to be a Fragment, since there's only one thing inside it:
```suggestion
  description: p([
    `The Global Parkinson's Genetics Program (GP2) aims to transform our understanding of the genetic 
    basis of Parkinson’s disease (PD) across diverse populations, including those underserved in 
    biomedical research. This program includes targeted collection and analysis of data and samples 
    from Africa, Asia, Europe and the Americas toward genetic insights that will broaden biomarker and 
    therapeutic discovery and development. GP2 is a resource of the Aligning Science Across 
    Parkinson’s (ASAP) initiative, which has developed an ambitious roadmap to tackle key scientific 
    challenges in PD by supporting meaningful, multidisciplinary collaboration; generating 
    research-enabling resources; and sharing data.`
  ])
```